cdp_scrapers/legistar_utils.py

```diff
@@ -761,6 +768,66 @@ def get_roles(
             ]
         )
 
+    def resolve_person_alias(self, person: Person) -> Optional[Person]:
+        """
+        If input person is in fact an alias of a reference known person,
+        return the reference person instead.
+        Else return person as-is.
+
+        Parameters
+        ----------
+        person: Person
+            Person to check whether is an alias or a real unique Person
+
+        Returns
+        -------
+        Person
+            input person, or the correct reference Person if input person is an alias.
+
+        See Also
+        --------
+        person_aliases in __init__()
+        """
+        if not self.person_aliases:
+            return person
+
```

Comment:
If I'm understanding this correctly -- if the `person.name in self.person_aliases`, you wouldn't need to go through this whole thing and just return `person`?

Reply:
:100: You are absolutely correct. Thank you.